avocado/core/job.py

```diff
@@ -240,10 +240,14 @@ def _setup_job_category(self):
         try:
             os.symlink(os.path.relpath(self.logdir, category_path),
                        os.path.join(category_path, os.path.basename(self.logdir)))
-        except Exception:
+        except NotImplementedError:
```

Comment:
I don't quite understand where the `NotImplementedError` can be raised from.  The most sensitive operation here seems to be `os.symlink`, which according to https://docs.python.org/3/library/os.html#os.symlink doesn't raise it.

Reply:
Yes, I read that and found out that on the older versions of windows it raise `NotImplementedError`. I do not really sure what is the support of avocado for such systems so I was thinking that will be better to have it there.